runtime/api/runtime.proto

```diff
@@ -11,6 +11,25 @@ option go_package = "github.com/rilldata/rill/runtime/api;api";
 // RuntimeService is a data infrastructure proxy and orchestrator based on Rill SQL.
 // It has a concept of instances, which are fully isolated data projects, enabling optional multi-tenancy.
 service RuntimeService {
+  // table api
+  rpc TableCardinality(CardinalityRequest) returns (CardinalityResponse) {
+    option (google.api.http) = {
+      get: "/v1/instances/{instance_id}/tables/{table_name}/cardinality"
+    };
+  }
+
+  rpc ProfileColumns(ProfileColumnsRequest) returns (ProfileColumnsResponse) {
+    option (google.api.http) = {
+      get: "/v1/instances/{instance_id}/tables/{table_name}/profile-columns"
+    };
+  }
+
+  rpc TableRows(RowsRequest) returns (RowsResponse) {
+    option (google.api.http) = {
+      get: "/v1/instances/{instance_id}/tables/{table_name}/rows"
+    };
+  }
+
```

Comment:
Maybe move these below the explore APIs as discussed in our meeting

Reply:
moved